seed/serializers/pint.py

```diff
@@ -137,7 +137,10 @@ def to_representation(self, obj):
             try:
                 org = state.organization
             except AttributeError:
-                org = state.state.organization
+                try:
+                    org = state.state.organization
+                except AttributeError:
+                    org = state.property_state.organization
```

Comment:
What's the story behind this change? Is `state` in L143 an AnalysisPropertyView object? Maybe this could just be a short clarifying comment in that last except block.

Reply:
Ted and I spent some time trying to figure this one out but yeah, the state is an AnalysisPropertyView object.  I can add a comment to denote this.